### ISSUE TITLE
wallet listener not aware of CHAIN_TYPE config

### ISSUE THREAD

**Issue by antiochp** [mimblewimble/grin#1479]
Running a wallet listener (or presumably _any_ wallet process) locally, unaware of the CHAIN_TYPE config (which lives in `grin-server.toml`, but not `grin-wallet.toml`) -

```
Sep 05 10:30:14.629 DEBG Using wallet seed file at: /antiochp/grin/node1/wallet_data/wallet.seed
************************ RwLockReadGuard { lock: RwLock { data: Mainnet } }
```

So we are not using the correct "coinbase maturity" configuration for a USER_TESTING chain locally.
And we use a value of `1,000` and not the expected value `3`...

**Comment by yeastplume** [comment]
understood, will add chain type to the wallet config later today

**Comment by yeastplume** [comment]
addressed in #1480
